# 工件信息

# 工艺类型
int32 type
# 工步数
int32 process_nb
# 工步信息
process[] all_process